packages/appium/docs/zh/quickstart/next-steps.md

```diff
@@ -1,6 +1,23 @@
 ---
-title: Next Steps
+hide:
+  - toc
+
+title: 下一步
 ---
 
+既然您已经成功配置了您的系统进行Android自动化并运行了一个简单的测试，
+您可能希望继续探索这份文档。特别是对于初学者，
+以下是一些很好的指南和参考资料：
+
+- 在 [生态系统](../ecosystem/index.md) 页面: 浏览可用的驱动程序, 客户端, 插件, 和工具
+- [管理Appium驱动程序与插件](../guides/managing-exts.md)
+- [能力](../guides/caps.md)
+- [设置](../guides/settings.md)
+
+您还会发现[Appium 监测者](https://github.com/appium/appium-inspector) 编写 Appium 测试不可或缺的工具。
+是编写 Appium 测试不可或缺的工具，因为它可以对应用程序进行可视化检查，并
```

Comment:
It looks like parts of this line are duplicated from the previous line?

Reply:
Yes，I  already   checked it, but in `packages/appium/docs/en/quickstart/next-steps.md`, that is same content.